content/en/tracing/service_catalog/guides/integrating-with-ci-visibility.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Integrating with CI Visibility
+kind: guide
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+---
+
+In the Software Delivery lens you can see data from your pipeline executions associated to your services. 
```

Comment:
In the Delivery lens, you can see CI pipeline and static analysis results associated to your services.